submitr/submission.py

```diff
@@ -690,7 +690,8 @@ def submit_any_ingestion(ingestion_filename, *,
                                  consortium=consortium, submission_center=submission_center)
 
     if not portal.ping():
-        SHOW(f"Portal credentials do not seem to work: {portal.keys_file} ({env})")
+        SHOW(f"Portal credentials do not seem to work: {portal.keys_file} ({env}). "
+             "Please double-check your credentials.")
```

Comment:
If you're updating this I'd make it more informative ie: log into the portal and check that your key is not expired, otherwise refresh the key.